src/components/modal/CdrModal.vue

```diff
@@ -263,21 +333,28 @@ const verticalSpace = computed(() => {
     : windowedSpace + fullscreenSpace;
   // fullscreen, here, would account for outerWrap padding, which is the same CdrSpaceTwoX
 });
-    const scrollMaxHeight = computed(() => totalHeight.value
-    - headerHeight.value
-    - verticalSpace.value);
+
+const scrollMaxHeight = computed(() => totalHeight.value
+  - headerHeight.value
+  - verticalSpace.value);
 
 const scrollPadding = computed(() => {
   const isScrolling = scrollHeight.value > offsetHeight.value;
   const hasScrollbar = offsetWidth.value - clientWidth.value > 0;
   if (isScrolling && hasScrollbar) {
     return 4;
-  } if (isScrolling) {
+  }
+  if (isScrolling) {
     return 12;
   }
   return 0;
 });
 
+const textContentStyle = computed(() => ({
+  maxHeight: `${scrollMaxHeight.value}px`,
+  paddingRight: `${scrollPadding.value}px`,
+}));
+
```

Comment:
Moved this out of template - was throwing a lint error for line length having it inline below.